recipe/meta.yaml

```diff
@@ -42,6 +42,9 @@ test:
     - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.dylib   # [osx]
     - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.so      # [linux]
     {% endfor %}
+    - test -L ${PREFIX}/lib/libtinfo.a
+    - test -L ${PREFIX}/lib/libtinfo.dylib                     # [osx]
+    - test -L ${PREFIX}/lib/libtinfo.so                        # [linux]
```

Comment:
Ensure these libraries not only exist (already done in the above statement), but that they are symlinks.